windows-active-directory-mixin/README.md

```diff
@@ -0,0 +1,111 @@
+# Windows Active Directory mixin
+
+The Windows Active Directory mixin is a set of configurable Grafana dashboards and alerts.
+
+The Windows Active Directory mixin contains the following dashboards:
+
+- Windows Active Directory overview
+- Windows Active Directory logs
+
+and the following alerts:
+- alertsCriticalHighReplicationIssues 
+- alertsWarningHighBindOperations
+- alertsWarningHighPasswordChanges
+- alertsCriticalMetricsDownJobName
+
+## Windows Active Directory overview
+
+The Windows Active Directory overview dashboard provides details on integration status/alerts, workers/coordinators, error failures, data throughput, blocked nodes, and distributed bytes.
+![Windows Active Directory overview dashboard (LDAP)](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/windows_active_directory_overview_1.png)
+![Windows Active Directory overview dashboard (database)](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/window_active_directory_overview_2.png)
+
+# Windows Active Directory logs
+
+The Windows Active Directory logs dashboard provides details on incoming system logs.
+![Windows Active Directory logs dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/windows_active_directory_logs.png)
+
+Windows Active Directory system logs are enabled by default in the `config.libsonnet` and can be removed by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for system logs ingested into your logs datasource, please also include the matching `instance` and `job` labels onto the [scrape configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
+
+```yaml
+    scrape_configs:
+      # Add here any snippet that belongs to the `logs.configs.scrape_configs` section.
+      # For a correct indentation, paste snippets copied from Grafana Cloud at the beginning of the line.
+    - job_name: integrations/windows
+      windows_events:
+        use_incoming_timestamp: true
+        eventlog_name: 'Application'
+        bookmark_path: "./bookmarks-app.xml"
+      relabel_configs:
+        - source_labels: ['computer']
+          target_label: 'agent_hostname'
+      pipeline_stages:
+        - json:
+            expressions:
+              source: source
+              level: levelText
+        - labels:
+            source:
+            level:
+            job: "integrations/windows"
```

Comment:
shouldn't we relabel the job to something like "integration/active_directory" or something like that to tell the difference between different windows jobs (like IIS)?

Reply:
that's interesting. because for IIS i thought we were changing the label to NOT be the generic "integrations/windows" so that we could filter by job and only have the IIS metrics. either way, if vitaly suggested that, then i'm ok with it as well